site/content/blog/2020-11-02-whats-the-deal-with-sveltekit.md

```diff
@@ -0,0 +1,89 @@
+---
+title: What's the deal with SvelteKit?
+description: We're rethinking how to build Svelte apps. Here's what you need to know
+author: Rich Harris
+authorURL: https://twitter.com/rich_harris
+---
+
+<aside><p>If you <em>didn't</em> attend Svelte Summit, you can catch up on the <a href="https://www.youtube.com/c/SvelteSociety/videos">Svelte Society YouTube page</a></p></aside>
+
+If you attended [Svelte Summit](https://sveltesummit.com/) last month you may have seen my talk, [Futuristic Web Development](https://www.youtube.com/watch?v=qSfdtmcZ4d0), in which I finally tackled one of the most frequently asked questions about Svelte: when will Sapper reach version 1.0?
+
+The answer: never.
+
+This was slightly tongue-in-cheek — as the talk explains, it's really more of a rewrite of Sapper coupled with a rebrand — but it raised a lot of new questions from the community, and it's time we offered a bit more clarity on what you can expect.
+
+
+## What's Sapper?
+
+[Sapper](https://sapper.svelte.dev) is an *app framework* (or 'metaframework') built on top of Svelte (which is a *component* framework). Its job is to make it easy to build Svelte apps with all the modern best practices like server-side rendering (SSR) and code-splitting, and to provide a project structure that makes development productive and fun. It uses *filesystem-based routing* (as popularised by [Next](https://nextjs.org/) and adopted by many other frameworks, albeit with some enhancements) — your project's file structure mirrors the structure of the app itself.
+
+While the Svelte homepage and documentation encourages you to [degit](https://github.com/Rich-Harris/degit) the [sveltejs/template](https://github.com/sveltejs/template) repo to start building an app, Sapper has long been our recommended way to build apps; this very blog post is (at the time of writing!) rendered with Sapper.
+
+
+## Why is it being wound down?
```

Comment:
Not the end of Sapper :wink: 

```suggestion
## Why is a successor being introduced?
```

Reply:
Isn't it? Are we planning to continue developing it once SvelteKit is released, beyond essential patches? I think it's important to set expectations so that people don't imagine they'll be able to continue using Sapper indefinitely

Reply:
I just meant that a replacement is being offered. It sounds more positive / less scary to say we're introducing a successor than we're winding down the thing you're using.

Reply:
Fair. Perhaps 'Why is it being replaced?'

Reply:
It's important to delineate expectations clearly, and comparing it to Sapper sets the wrong expectation. `Svelte#Kit` is different enough from its predecessor that it really is something new. It's enough to say the upgrade path should be relatively simple.

Reply:
Or maybe "Why is it being rewritten?" or "Why is it being rebranded?". Those are the terms used in the first paragraph, which sounded fine to me. "replaced" still sounds slightly on the scary side, though I do prefer it to "wound down". I'm trying to be super sensitive here because of all the very worried people I've seen on Discord when they've heard just a small snippet of what's happening and haven't looked into the details

Reply:
just saw arx's reply - "Why are we migrating?" would also be fine. just trying to find the happiest sounding words here for people that see the headings and skip the text :smiley:

Reply:
I like that — have updated to 'Why are we migrating to something new?'